docs/docs/apis/core/account-api.md

```diff
@@ -0,0 +1,216 @@
+
+// Alternative: Direct signing with account
+Transaction signedTx = account.sign(transaction);
+```
+
+## Advanced Usage
+
+### Multi-Account Management
+
+Manage multiple accounts for different purposes:
+
+```java
+// Main spending account
+Account spendingAccount = new Account(Networks.mainnet(), spendingMnemonic);
+
+// Savings account
+Account savingsAccount = new Account(Networks.mainnet(), savingsMnemonic);
+
+// Business account
+Account businessAccount = new Account(Networks.mainnet(), businessMnemonic);
+
+// Get addresses for each account
+String spendingAddress = spendingAccount.baseAddress();
+String savingsAddress = savingsAccount.baseAddress();
+String businessAddress = businessAccount.baseAddress();
+```
+
+### Network-Specific Operations
+
+Handle different network configurations:
+
+```java
+// Mainnet account
+Account mainnetAccount = new Account(Networks.mainnet(), mnemonic);
+String mainnetAddress = mainnetAccount.baseAddress(); // addr1...
+
+// Testnet account
+Account testnetAccount = new Account(Networks.testnet(), mnemonic);
+String testnetAddress = testnetAccount.baseAddress(); // addr_test1...
+
+// Preview network account
+Account previewAccount = new Account(Networks.preview(), mnemonic);
+String previewAddress = previewAccount.baseAddress(); // addr_test1...
+```
+
+## Best Practices
+
+### Secure Key Storage
+
+```java
+// Generate secure mnemonic
+Account account = new Account(Networks.mainnet());
+String mnemonic = account.mnemonic();
+
+// Store mnemonic securely (not in plain text)
+// Use secure storage mechanisms in production
+```
+
+### Mnemonic Validation
+
+```java
+// Validate mnemonic before creating account
+public boolean isValidMnemonic(String mnemonic) {
+    try {
+        new Account(Networks.testnet(), mnemonic);
+        return true;
+    } catch (Exception e) {
+        return false;
+    }
+}
+```
+
+### Address Verification
+
+```java
+// Verify account generates expected addresses
+Account account = new Account(Networks.testnet(), knownMnemonic);
+String expectedAddress = "addr_test1..."; // Known address for this mnemonic
+
+if (account.baseAddress().equals(expectedAddress)) {
+    System.out.println("Account verification successful");
+} else {
+    System.out.println("Account verification failed");
+}
+```
+
+## Error Handling
+
+Handle common account-related errors:
+
+```java
+try {
+    // Create account with potentially invalid mnemonic
+    Account account = new Account(Networks.mainnet(), userMnemonic);
+    String address = account.baseAddress();
+} catch (Exception e) {
+    if (e.getMessage().contains("Invalid mnemonic")) {
+        System.err.println("Mnemonic phrase is invalid");
+    } else if (e.getMessage().contains("Network")) {
+        System.err.println("Network configuration error");
+    } else {
+        System.err.println("Account creation failed: " + e.getMessage());
+    }
+}
+```
+
+## Account vs HD Wallet Comparison
+
+| Feature | Account API | HD Wallet API |
```

Comment:
I think we can remove this section. The main difference is 

1. Account Api deals with a single account. 
2. HDWallet api is a wrapper around Account api and can handle multiple accounts generated from the same mnemonic.

Let's remove this section for now.